getting-started/multiband-analysis.rst

```diff
@@ -63,39 +70,44 @@ From the source table's schema you know this flux has units of counts:
+   rCoaddPhotoCalib = butler.get('deepCoadd_calexp.photoCalib', band='r', tract=9813, patch=41)
+   iCoaddPhotoCalib = butler.get('deepCoadd_calexp.photoCalib', band='i', tract=9813, patch=41)
 
 .. note::
 
    An alternative way to get the ``lsst.afw.image.PhotoCalib`` object is from the ``deepCoadd_calexp`` dataset object:
 
    .. code-block:: python
 
-      rCoaddCalexp = butler.get('deepCoadd_calexp',  {'filter': 'HSC-R', 'tract': 0, 'patch': '1,1'})
+      rCoaddCalexp = butler.get('deepCoadd_calexp', band='r', trct=9813, patch=41)
       rCoaddPhotoCalib = rCoaddCalexp.getPhotoCalib()
 
+   Note that this method is slower than getting just the ``photoCalib`` component and should only be used if you intend to use the ``calexp`` later on.
+
 These ``PhotoCalib`` objects not only have methods for directly accessing calibration information, but also for applying those calibrations.
-Use the ``PhotoCalib.instFluxToMagnitude()`` method to transform instrumental fluxes in counts to AB magnitudes, and ``PhotoCalib.instFluxToNanojanksy()`` to transform counts into nanojansky. When called with an ``lsst.afw.table.SourceCatalog`` and string specifying the flux field name, these methods each return an array with the magnitude and magnitude error as columns.
+Use the ``PhotoCalib.instFluxToMagnitude()`` method to transform instrumental fluxes in counts to AB magnitudes, and ``PhotoCalib.instFluxToNanojanksy()`` to transform counts into nanojansky.
+When called with an ``lsst.afw.table.SourceCatalog`` and string specifying the flux field name, these methods each return an array with the magnitude and magnitude error as a list of tuples.
 
 .. code-block:: python
 
+   gMags = gCoaddPhotoCalib.instFluxToMagnitude(gSources, 'base_PsfFlux')
    rMags = rCoaddPhotoCalib.instFluxToMagnitude(rSources, 'base_PsfFlux')
-   iMags = iCoaddPhotoCalib.instFluxToMagnitude(iSources, 'base_PsfFlux'])
+   iMags = iCoaddPhotoCalib.instFluxToMagnitude(iSources, 'base_PsfFlux')
 
 Filtering for unique, deblended sources with the detect_isPrimary flag
 ======================================================================
 
 Before going ahead and plotting a CMD from the full source table, you'll typically need to do some basic filtering.
 Exactly what filtering is done depends on the application, but source tables should *always* be filtered for unique sources.
 There are two ways that measured sources might not be unique: deblended sources and sources in patch overlaps.
-Additionally, some sources are "sky" objects added by ``detectCoaddSources.py`` for noise characterization that you need to filter out.
+Additionally, some sources are "sky" objects added by the coadd measurement step for noise characterization that you need to filter out.
 This section gives a brief introduction to removing duplicate and unwanted sources, for details see :doc:`/modules/lsst.pipe.tasks/deblending-flags-overview`.
```

Comment:
This link is broken

Reply:
I've verified that in a full build with all packages included, this link does resolve